### PR TITLE
Adding vortex.dino.icu

### PR DESCRIPTION
**Pull request by SkyfallWasTaken**
# Adding `vortex.dino.icu`, `api.vortex.dino.icu`, `mail.vortex.dino.icu`

## Description

Firstly, I apologise if the format is incorrect, I wasn't sure how to add multi level subdomains.

[Vortex](https://vortex-email.vercel.app) is a disposable email service I'm running. The subdomain would be used for:
- The frontend
- The SMTP server
- The backend API

Thanks!

### CODE REVIEW

**Review by alx-alexpark — COMMENTED** [review]
Hi!!

Stuff looks good, but before letting people send emails, I want to confirm with @grymmy.

File: dino.icu.yaml [hunk 1]
```diff
@@ -403,7 +403,36 @@ vercel-ns:
...
+- ttl: 600
+  type: MX
+  values:
+  - exchange: mail.vortex.dino.icu.
+    preference: 10
+
+- ttl: 600
+  type: TXT
+  values:
+  - v=spf1 ip4:35.193.1.201 ~all
```
Comment (Muirrum) [inline dino.icu.yaml:423]: fix indentation

```suggestion
  - ttl: 600
    type: A
    value: 76.76.21.21

  - ttl: 600
    type: MX
    values:
      - exchange: mail.vortex.dino.icu.
        preference: 10

  - ttl: 600
    type: TXT
    values:
      - v=spf1 ip4:35.193.1.201 ~all
```

**Review by Muirrum — APPROVED** [review]
disregard, github web editor masked the indentation

**Comment by SkyfallWasTaken** [comment]
Thanks!

**Pull request merged**